composer.json

```diff
@@ -6,10 +6,10 @@
     "license": "BSD-3-Clause",
     "require": {
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-cms": "^4.5@dev",
-        "dnadesign/silverstripe-elemental": "4.2.x-dev",
-        "silverstripe/elemental-fileblock": "2.0.x-dev",
-        "silverstripe/elemental-bannerblock": "2.0.x-dev"
+        "silverstripe/recipe-cms": "^4",
```

Comment:
I think this should still be `^4@dev`? Otherwise it will track the latest stable, rather than the next minor?

Reply:
done

Reply:
I think `^4@dev` wouldn't override `prefer-stable`. It would rather allow installing dev and we already have minimum-stability `dev` anyway, so that won't have any effect in this context

Reply:
Hmm, I'm still not sure about this change. The current convention is that the branch fetches dev dependencies (e.g. points to dev branches), while tagged releases point to stable tags. I think it's still useful to be able to create projects from branches of a recipe - the recipe most commonly used in this way is kitchen-sink, but it's useful for all of them.

This also may have a flow-on effect on the requirements determined during an install of kitchen-sink's `2` branch - which absolutely should not point to stable releases instead of dev branches, as it's supposed to test the latest changes.

Reply:
(Had a chat offline - keeping the dependency as `^4` allows for both stable and dev versions to be installed based on external requirements, so this is the best way forward.)